packages/design-system/src/components/N8nActionDropdown/ActionDropdown.vue

```diff
@@ -93,12 +93,10 @@ export default Vue.extend({
 			this.$emit('select', action);
 		},
 		onButtonBlur(event: FocusEvent): void {
-			const elementDropdown = this.$refs.elementDropdown;
+			const elementDropdown = this.$refs.elementDropdown as Vue & { hide: () => void };
```

Comment:
Should this intersection be unionized to `undefined`? The ref'd element might not exist, which is why we're checking for it at L98.

Reply:
If there is no such element it will stay `undefined`, won't break. The only problem I see here is if the ref is placed on a wrong element (in this case, calling `hide` method will fail if the component doesn't contain such method). I considered this a case of wrong component usage but I can check if we are dealing with `ElDropdown` before calling the function.

Reply:
I mention this because I notice many of the components that use refs e.g. `Select.vue` follow this approach of asserting the type of the ref'd element as `Vue & SomeType | undefined`, which makes sense given that the ref might be missing, so that we remember to check for it.

Definitely a minor detail and not a blocker for merging.